data/gegno/gegno I corroboration.txt

```diff
@@ -0,0 +1,1178 @@
+	source "Giaru Gegno"
+	to offer
+		has "Gegno Suspicions: declined"
+		not "Gegno Genocide Defense: offered"
+		not "yvx killed"
+	to complete
+		never
+	on enter "Aleit"
+		fail
+
+	npc
+		on kill
+			set "yvx killed"
+		government "Gegno Vi"
+		personality heroic vindictive ramming daring entering fleeing uninterested
+		system "Aleit"
+		ship "Rhyolite" "Imtydei"
+
+
+mission "Scin Surveillance"
+	invisible
+	repeat
+	landing
+	deadline 7
+	source
+		near "Heutesl" 100
+	to offer
+		random < 70
+		has "event: gegno: suspicions"
+		not "Scin Suspicion: offered"
+		not "Gegno Genocide Defense: offered"
+		not "Gegno Anticipation: offered"
+		not "Gegno Intervention: offered"
+	to complete
+		never
+	npc save
+		government "Gegno Scin"
+		personality surveillance
+		system
+			government "Gegno Scin"
+		fleet
+			names "gegno scin"
+			variant
+				"Tridymite"
+
+
+mission "Veering Vi A"
+	repeat
+	invisible
+	landing
+	deadline 7
+	to offer
+#		random < 30
+		has "Gegno Suspicions: declined"
+		not "Full Vi Backlash: offered"
+		not "Gegno Genocide Defense: offered"
+		not or
+			"Gegno Anticipation: offered"
+			"Gegno Intervention: offered"
+	npc
+		on kill
+			"vi backlash" ++
+		system "Ookmya"
+		government "Gegno Vi (Combative)"
+		personality heroic staying
+		fleet
+			names "gegno vi"
+			variant 10
+				"Dunite"
+			variant
+				"Dunite" 2
+
+
+mission "Veering Vi B"
+	repeat
+	invisible
+	landing
+	deadline 7
+	to offer
+#		random < 30
+		has "Gegno Suspicions: declined"
+		not "Full Vi Backlash: offered"
+		not "Gegno Genocide Defense: offered"
+		not or
+			"Gegno Anticipation: offered"
+			"Gegno Intervention: offered"
+	npc
+		on kill
+			"vi backlash" ++
+		system
+			government "Gegno Vi"
+		government "Gegno Vi (Combative)"
+		personality heroic staying
+		fleet
+			names "gegno vi"
+			variant
+				"Eclogite"
+
+
+mission "Full Vi Backlash"
+	invisible
+	landing
+	to offer
+		"vi backlash" >= 3
+		"reputation: Gegno Vi" >= 0
+	on offer
+		fail
+		clear "vi backlash"
+		"reputation: Gegno Vi" = -1
+
+
+mission "Return to Giaru Gegno"
+	landing
+	description `The Quarg known as Adrauni has summoned you back to <destination> to discuss their negotiations with the Gegno.`
+	source
+		not planet "Giaru Gegno"
+	destination "Giaru Gegno"
+	to offer
+		has "event: gegno: quarg talks"
+		"gegno: encounters" < 6
+		not "Gegno Genocide Defense: offered"
+	on enter
+		dialog
+			`Upon departure from <origin>, you receive an incoming message. "Greetings, human <first> <last>. This message is from Adrauni. Please return to our ringworld at the first opportunity given to you."`
+				accept
+
+
+mission "Gegno Anticipation"
+	landing
+	description "Visit <destination>."
+	source "Giaru Gegno"
+	destination "Dueyu Eitch"
+	clearance
+	to offer
+		has "Return to Giaru Gegno: offered"
+		not "Gegno Genocide Defense: offererd"
+		not "Gegno Intervention: offered"
+		not or
+			"reputation: Gegno Vi" <= -100
+			"reputation: Gegno Scin" <= -100
```

Comment:
```suggestion
		"reputation: Gegno Vi" > -100
		"reputation: Gegno Scin" > -100
```
there's this one and another one above too (mb you're already fixing it offline tho)

Reply:
~~Yep, got some others too.~~